bootc/Containerfile.centos9

```diff
@@ -4,34 +4,64 @@ RUN rm -rf /etc/yum.repos.d/*.repo
+	iptables-services \
+	iscsi-initiator-utils \
+	jq \
+	lvm2 \
+	NetworkManager-ovs \
+	nftables \
+	numactl \
+	openssh-server \
+	openstack-selinux \
+	openvswitch \
+	os-net-config \
+	podman \
+	python3-libselinux \
+	python3-pyyaml \
+	rsync \
+	sysstat \
+	tmpwatch \
+	tuned-profiles-cpu-partitioning"
+
 ARG ENABLE_UNITS="openvswitch"
 
 RUN dnf -y update && dnf -y install $PACKAGES && dnf clean all && systemctl enable $ENABLE_UNITS
+
+# Template systemd service for services
+COPY embedded-services/quadlets/systemd/service-template.kube /usr/share/containers/systemd/edpm-compute@.kube
+
+## Service specific quadlets
+COPY embedded-services/quadlets/ovn-controller/ovn_controller.yaml /usr/share/containers/systemd/ovn_controller.yaml
+COPY embedded-services/quadlets/ovn-controller/ovn_controller.image /usr/share/containers/systemd/ovn_controller.image
+COPY embedded-services/quadlets/iscsid/iscsid.yaml /usr/share/containers/systemd/iscsid.yaml
+COPY embedded-services/quadlets/iscsid/iscsid.image /usr/share/containers/systemd/iscsid.image
+COPY embedded-services/quadlets/nova_compute/nova_compute.yaml /usr/share/containers/systemd/nova_compute.yaml
+COPY embedded-services/quadlets/nova_compute/nova_compute.image /usr/share/containers/systemd/nova_compute.image
+COPY embedded-services/quadlets/ovn_metadata_agent/ovn_metadata_agent.yaml /usr/share/containers/systemd/ovn_metadata_agent.yaml
+COPY embedded-services/quadlets/ovn_metadata_agent/ovn_metadata_agent.image /usr/share/containers/systemd/ovn_metadata_agent.image
+COPY embedded-services/quadlets/logrotate_crond/logrotate_crond.yaml /usr/share/containers/systemd/logrotate_crond.yaml
+COPY embedded-services/quadlets/logrotate_crond/logrotate_crond.image /usr/share/containers/systemd/logrotate_crond.image
+COPY embedded-services/quadlets/multipathd/multipathd.yaml /usr/share/containers/systemd/multipathd.yaml
+COPY embedded-services/quadlets/multipathd/multipathd.image /usr/share/containers/systemd/multipathd.image
+COPY embedded-services/quadlets/ceilometer_agent_compute/ceilometer_agent_compute.yaml /usr/share/containers/systemd/ceilometer_agent_compute.yaml
+COPY embedded-services/quadlets/ceilometer_agent_compute/ceilometer_agent_compute.image /usr/share/containers/systemd/ceilometer_agent_compute.image
+
+# Pre-cache containers for each service
+RUN podman pull quay.io/podified-antelope-centos9/openstack-ceilometer-compute:current-podified
+RUN podman pull quay.io/podified-antelope-centos9/openstack-iscsid:current-podified
+RUN podman pull quay.io/podified-antelope-centos9/openstack-cron:current-podified
+RUN podman pull quay.io/podified-antelope-centos9/openstack-multipathd:current-podified
+RUN podman pull quay.io/podified-antelope-centos9/openstack-nova-compute:current-podified
+RUN podman pull quay.io/podified-antelope-centos9/openstack-ovn-controller:current-podified
+RUN podman pull quay.io/podified-antelope-centos9/openstack-neutron-metadata-agent-ovn:current-podified
```

Comment:
Do we want each pull in its own layer? I don't know?? If we didn't then this could be reduced to a single RUN

Reply:
Same for this one

Reply:
This actually isn't working anymore. Complains about things not being available in the container environment. I'm actually noticing the same error with my laptop bootc image too. Maybe something changed in the base edpm centos image. For now, I'm just commenting these lines when I build the image and letting `systemd` pull the containers when the node boots.